modules/rosa-sdpolicy-account-management.adoc

```diff
@@ -49,7 +49,9 @@ Control plane and infrastructure nodes are deployed and managed by Red Hat. Shut
 
 [NOTE]
 ====
-1 vCPU core and 1 GiB of memory are reserved on each worker node to run processes required as part of the managed service. This includes, but is not limited to, audit log aggregation, metrics collection, DNS, image registry, and SDN.
+Approximately 1 vCPU core and 1 GiB of memory are reserved on each worker node and removed from allocatable resources. This allocation of resources is necessary to run processes required by the underlying platform. These processes include system daemons such as udev, kubelet, and container runtime among others. It also accounts for kernel reservations. {OCP} core systems such as audit log aggregation, metrics collection, DNS, image registry, SDN, and others might consume additional allocatable resources to maintain the stability and maintainability of the cluster. The additional resources consumed might vary based on usage.
```

Comment:
> Approximately 1 vCPU core and 1 GiB of memory are reserved on each worker node and removed from allocatable resources. This allocation of resources is necessary to run processes required by the underlying platform.

There is nothing wrong with this, but I wonder if s/allocation/reservation might make it more clear which pool is which since "allocatable" is referring to the non-reserved resources.